.github/workflows/python-package.yml

```diff
@@ -11,32 +11,31 @@ on:
 
 jobs:
   build:
+  
+   runs-on: ${{ matrix.os }}
+   strategy:
+     matrix:
+      os: [ubuntu-latest]
+      python-version: [3.6, 3.7, 3.8, 3.9]
 
-    runs-on: [ubuntu-latest, windows-latest, macos-latest]
-    strategy:
-      matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
+   steps:
+     - uses: actions/checkout@v2
+     - name: Set up Python ${{ matrix.python-version }}
+       uses: actions/setup-python@v2
+       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: |
+     - name: Install dependencies
```

Comment:
Remove additional space

Reply:
it is causing a syntax error in the yaml format. To verify if your yaml is correct, you can copy paste
the contents of the file into https://codebeautify.org/yaml-validator/

it will tell you if it is valid yaml

Reply:
all "- name" entities must lie at the same indentation level, so you need to remove all additional spaces that you have added. You needed to change the spacing only for the `env` variable at the bottom of the file.

Reply:
corrected indentation